packages/settingeditor-extension/src/index.ts

```diff
@@ -45,6 +50,88 @@ namespace CommandIDs {
  * The default setting editor extension.
  */
 const plugin: JupyterFrontEndPlugin<ISettingEditorTracker> = {
+  id: '@jupyterlab/settingeditor-extension:simple-plugin',
```

Comment:
We need a better name.

```suggestion
  id: '@jupyterlab/settingeditor-extension:form-ui',
```

Reply:
Agreed - any ideas?